databases/week1/homework.sql

```diff
@@ -0,0 +1,12 @@
+select * from task
+select count(title) from `task`;
+select count(title) from `task` where due_date is null;
+select title from task where updated <= due_date
+select title from task where updated > due_date
+select title from task where status_id=3;
```

Comment:
Let's assume you don't know the status_id but you have a status="Done" which can also be used to get a result. You could do something like this:
`SELECT 
  * 
FROM 
  task 
  JOIN status ON task.status_id = status.id 
WHERE 
  status.name = "done";`